fs/void-musl/info

```diff
@@ -0,0 +1,17 @@
+#!/bin/false
+
+# basic info
+FS_FANCYNAME="Void-musl"
+#FS_HAS_OOBE="true"
+FS_INST_PKG="/bin/xbps-install -y"
+FS_UPDATE="/bin/xbps-install -Syu"
+
+# bootloader packages
+FS_PKGS_CD_BOOTFW_UBOOT="u-boot-tools curl wget"
+FS_PKGS_CD_BOOTFW_DEPTHCHARGE="$FS_PKGS_CD_BOOTFW_UBOOT vboot-utils"
+
+# ui packages
```

Comment:
get those working if you can

Reply:
graphical environments require extra configuration on void that isn't done in advance on any other void installer, and would require a lot of void-specific code in install script.